### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by anandjangalicsg**

### CODE REVIEW

File: src/main/java/com/hrbp/feedback/config/Constants.java
Comment (sukanta-biscred) [inline src/main/java/com/hrbp/feedback/config/Constants.java:1]: Please remove the constructor.

File: src/main/java/com/hrbp/feedback/controller/FeedbackController.java
Comment (sukanta-biscred) [inline src/main/java/com/hrbp/feedback/controller/FeedbackController.java:1]: route only will be feedback .
Only keep @RequestMapping("/api") here and feedback in individual routes like
@PostMapping("feedback")
@PutMapping("feedback") etc...

File: src/main/java/com/hrbp/feedback/controller/LoginController.java
Comment (sukanta-biscred) [inline src/main/java/com/hrbp/feedback/controller/LoginController.java:1]: Login controller will only be for Login all dashboard will be in a different controller

File: src/main/java/com/hrbp/feedback/exceptions/ResourceNotFoundException.java
Comment (sukanta-biscred) [inline src/main/java/com/hrbp/feedback/exceptions/ResourceNotFoundException.java:1]: What about other exceptions ??

**Review by sukanta-biscred — CHANGES_REQUESTED** [review]

File: src/main/java/com/hrbp/feedback/config/Constants.java
Comment (anandjangalicsg) [reply src/main/java/com/hrbp/feedback/config/Constants.java:1]: I have removed

**Pull request merged**